cloudsql/postgres/database-sql/connect_connector.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC
```

Comment:
nit: License headers should aim to reflect the year the file was originally created, updating the file does not require the license header to be changed.
```suggestion
// Copyright 2022 Google LLC
```

Reply:
Happy to switch these back. I had a different PR rejected for not having the current year. Is this a repository-by-repository choice? https://github.com/GoogleCloudPlatform/cloud-sql-proxy/runs/10259552138

Reply:
If the file already exists, then the older year takes precedence. If it's a new file, then naturally the current year is the way to go.

Reply:
Perfect. Makes total sense. Thank you!